### ISSUE TITLE
At least a Typo...

### ISSUE THREAD

**Issue by ChrisAtkinsonAL** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#1517]
Above it says:
Enable-CsMeetingRoom -Identity $rm -RegistrarPool'sippoolbl20a04.infra.lync.com' -SipAddressType EmailAddress

At a minimum, this is missing a space... -RegistrarPool'sippoolbl20a04.infra.lync.com' should be -RegistrarPool 'sippoolbl20a04.infra.lync.com'

Anddddd when the space is added, it gives an error about moving... and this error is not well documented.  I'm stuck.
Cannot move user in enable operation. Use the Move user cmdlet instead.
    + CategoryInfo          : InvalidOperation: (Enable-CsMeetingRoom:String) [Enable-CsMeetingRoom], InvalidOperationException
    + FullyQualifiedErrorId : ProcessRecord,Microsoft.Rtc.Management.AD.Cmdlets.EnableOcsMeetingRoomCmdlet
    + PSComputerName        : admin1a.online.lync.com

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9978e48b-5b8d-df59-d8d3-d3c90a8efafb
* Version Independent ID: 52429f1d-39a1-49fb-c9dd-d4bafa19760d
* Content: [Deploy Skype Room Systems v2 with Exchange Online](https://docs.microsoft.com/en-us/skypeforbusiness/deploy/deploy-clients/with-exchange-online)
* Content Source: [Skype/SfbServer/deploy/deploy-clients/with-exchange-online.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Skype/SfbServer/deploy/deploy-clients/with-exchange-online.md)
* Product: **skype-for-business-itpro**
* GitHub Login: @jambirk
* Microsoft Alias: **jambirk**

**Comment by ChrisAtkinsonAL** [comment]
Note that I do understand that I need to find another user and change my pool to match them, which I did....

**Comment by scanum** [comment]
@davgroom, @sohailta any updates on this issue? Thanks.

**Comment by jambirk** [comment]
@ChrisAtkinsonAL we recently refreshed that page with some info from the Exchange doc set, and your typo was fixed along the way. Are you still seeing this problem?

**Comment by ChrisAtkinsonAL** [comment]
I can see the typo is fixed, which is great.
The error I had related to moving is because new users seem to be created in the correct SIPPool these days, so enabling it and specifying a New pool equal to the value of the Current pool fails the "move operation".  So, it's worth noting that you don't need to specify "-sippool $sippool -sipaddresstype emailaddress" if your new user's sippool already matches the other users in your organization.  That's why I was getting the move error.  Once I removed those paramaters, viola!
